apps/angular/projection/src/app/data-access/city.store.ts

```diff
@@ -1,23 +1,22 @@
-import { Injectable } from '@angular/core';
-import { BehaviorSubject } from 'rxjs';
+import { Injectable, WritableSignal, signal } from '@angular/core';
 import { City } from '../model/city.model';
 
 @Injectable({
   providedIn: 'root',
 })
 export class CityStore {
-  private cities = new BehaviorSubject<City[]>([]);
-  cities$ = this.cities.asObservable();
+  cities: WritableSignal<City[]> = signal([]);
+  cities$ = this.cities.asReadonly();
```

Comment:
I'm not very fond of the $ for signal, since it looks like an observable, but that's my opinion, otherwise this is great 👍